source/collection-management/tests/timeseries-collection.yml

```diff
@@ -67,7 +68,11 @@ tests:
         object: *database0
         arguments:
           collection: *collection0Name
-          timeseries: *timeseries0
+          expireAfterSeconds:
+            $numberLong: "604800"
+          timeseries: &timeseries0
```

Comment:
I'm not sure if you intended to redefine this anchor, but I assume not since you're using the name numbering as the previous (zero). This is apparently [valid to do](https://stackoverflow.com/a/40376576); however, we don't redefine anchors in other tests AFAIK.

In this particular case, the value is exactly the same as the previous usage, so I would expect you could just replace this line with `timeseries: *timeseries0` and call it a day.

Reply:
Ah I did not, but good to know about yaml anchor re-definitions 🤔 

Just used `*timeseries0` here.